### ISSUE TITLE
docker image for arch other than amd64

### ISSUE THREAD

**Issue by contrun** [etesync/etesync-dav#215]
Currently we only have docker image for linux/amd64 https://hub.docker.com/r/etesync/etesync-dav/tags?page=1&ordering=last_updated.

**Comment by tasn** [comment]
Can I build for multiple destinations from amd64? Or alternatively, any idea how to easily build the images from CI?

**Comment by contrun** [comment]
Yeah. We can build images for multi architectures by buildx. Here is an example https://github.com/victor-rds/docker-etebase/blob/master/.github/workflows/publish-alpine.yml  . Here is the results https://hub.docker.com/r/victorrds/etesync/tags?page=1&ordering=last_updated

**Comment by tasn** [comment]
As always, @victor-rds to the rescue. :)

@victor-rds, what are all of those inline actions you have in the repo? How hard would it be to make etesync-dav also build for multiple platforms?

**Comment by victor-rds** [comment]
Just some helpers `get-data` is to generate multiple tag names for the same image, at the moment the `:slim` and `:0.7.0-slim` are the same image, the action just detect if the trigger event is a git tag and add versioned tags.

As for the `get-labels` is just for adding some labels, completely optional.

**Comment by victor-rds** [comment]
Just found this action that do almost the same thing as mine (and are better at) https://github.com/docker/metadata-action

**Comment by tasn** [comment]
Thanks! I'll try to figure it out and add it here.
